include/deal.II/distributed/tria.h

```diff
@@ -169,19 +169,19 @@ namespace parallel
      * {
      *   ... create the coarse mesh ...
      *
-     *   coarse_grid.signals.post_refinement.connect(
-     *     std::bind (&set_boundary_ids<dim>, std::ref(coarse_grid)));
+     *   coarse_grid.signals.post_refinement.connect([&coarse_grid](){
+     *     set_boundary_ids<dim>(coarse_grid);});
      * }
      * @endcode
      *
-     * What the call to <code>std::bind</code> does is to produce an
-     * object that can be called like a function with no arguments. It does so
-     * by taking the address of a function that does, in fact, take an
-     * argument but permanently fix this one argument to a reference to the
-     * coarse grid triangulation. After each refinement step, the
-     * triangulation will then call the object so created which will in turn
-     * call <code>set_boundary_ids<dim></code> with the reference to the
-     * coarse grid as argument.
+     * The code passed as argument to <code>connect</code> is an object
+     * that can be called like a function with no arguments. It does so by
+     * wrapping a function that does, in fact, take an argument but
+     * permanently fix this one argument to a reference to the coarse grid
```

Comment:
This second sentence here no longer represents what we actually do. It still speaks of binding arguments, but that's not how lambda functions should be thought of. Can you rephase?